installing/installing_bare_metal/installing-bare-metal.adoc

```diff
@@ -8,6 +8,15 @@ toc::[]
 In {product-title} version {product-version}, you can install a cluster on
 bare metal infrastructure that you provision.
 
+[IMPORTANT]
+====
+While you might be able to follow this procedure to deploy a cluster on
+virtualized or cloud environments, you must be aware of additional
+considerations for non-bare metal platforms. Review the information in the
+following
+link:https://access.redhat.com/articles/4207611[customer support article]
```

Comment:
This is personal preference, but I think it would make more sense to include the title of the article or an indication of what the article contains.

What about:

> Review the link:https://access.redhat.com/articles/4207611[guidelines for deploying on non-tested platforms] before you...

Reply:
I usually agree, but I question whether including "non-tested" in the link will make someone less likely to click it.